app/src/main/java/com/scrat/everchanging/TextureManager.java

```diff
@@ -77,22 +75,23 @@ float dipToPixels(final float dipValue) {
             GLES20.glBindTexture(GLES20.GL_TEXTURE_2D, textureID);
             textures[counter] = new Texture();
             textures[counter].textureID = textureID;
-            textures[counter].textureName = bothTextureNameList[counter];
-            @SuppressLint("DiscouragedApi") int drawableID = context.getResources().getIdentifier(textures[counter].textureName, "drawable", context.getPackageName());
-            Bitmap picture = null;
-            @SuppressLint("UseCompatLoadingForDrawables") Drawable drawable = context.getDrawable(drawableID);
-            final BitmapFactory.Options options = new BitmapFactory.Options();
+            textures[counter].textureResId = bothTextureResIdsList[counter];
+
+            final Bitmap picture;
+            final Drawable drawable = context.getDrawable(textures[counter].textureResId);
             if (drawable instanceof BitmapDrawable) {
-                picture = BitmapFactory.decodeResource(context.getResources(), drawableID, options);
-                textures[counter].width = picture.getWidth() / dipToPixels(1);
```

Comment:
dipToPixels is not needed now.

Reply:
dipToPixels  it's still necessary. Otherwise, the backgrounds fall apart, and the sizes of the "Crystals" and other rasters bitmaps do not match the original. 
<img width="494" alt="image" src="https://github.com/SCratORS/Everchanging/assets/24729719/b7797bff-f5a6-48bf-8bc9-fc816a0fec4a">

Reply:
Sorry, I tested it on multiple devices and didn't see the issue. But I had this issue before I removed dpToPixels 🤔🤔🤔

Reply:
Did you test after my commit or are these screnshots old?
In my commit, it does not work correctly with `dpToPixels`, #41

Reply:
After commit. I remember these problems, so immediately using a screen size of 480*800 I tested this commit. the result is higher. Then I just added dipToPixels(1) and everything became normal.